guidelines/index.html

```diff
@@ -828,8 +830,18 @@
 					<p class="change">Unchanged</p>
 					<p>For <a>Web pages</a> that require the user to submit information, at least one of the following is true:</p>
 				</section>
+        
+        <section class="sc">
+					<h4>Minimize user errors</h4>
+					<p class="conformance-level">A</p>
+					<p>Input error are automatic corrected where the correction can be reliably made.</p>
```

Comment:
Change "error" to "errors" and "correction" to "corrections" to match the verb "are"